_posts/2020-03-24-hhvm-4.50.markdown

```diff
@@ -0,0 +1,41 @@
+---
+title: "HHVM 4.50"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.50 is released! This release marks the end of support for 4.43;
+HHVM 4.45&ndash;4.49 remain supported, as do the 4.8 and 4.32 LTS releases.
```

Comment:
what happened to HHVM 4.44?

Reply:
It was the oldest in last week’s announcement, and this leaves us with 6 concurrent releases; am I missing something?

Reply:
It says end of support for 4.43 and 4.45-... remain supported, so 4.44 was missing.